examples/goapp/kv/app.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"embed"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/mattermost/mattermost-plugin-apps/apps/goapp"
+)
+
+// static is preloaded with the contents of the ./static directory.
+//go:embed static
+var static embed.FS
+
+// main starts the app, as a standalone HTTP server. Use $PORT and $ROOT_URL to
+// customize.
+func main() {
+	// Create the app, add `send` to the app's command and the channel header.
+	goapp.MakeAppOrPanic(
```

Comment:
Unrelated to this PR: The naming pattern usually used in Go for functions that panic on error is `MustX`. Should we adopt that for `goapp`?